CHANGELOG.md

```diff
@@ -2,6 +2,106 @@
 
 ## Unreleased
 
+### Migration notes
+
+- Apache Pulsar instrumentation name is changed from `io.opentelemetry.apache-pulsar-2.8` to
+  `io.opentelemetry.pulsar-2.8` and Apache Camel instrumentation name is changed from
+  `io.opentelemetry.apache-camel-2.20` to `io.opentelemetry.camel-2.20`
+  ([#8195](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8195))
+- Mojarra instrumentation suppression key has changed from `mojarra` to `jsf-mojarra`
+  and MyFaces instrumentation suppression key has changed from `myfaces` to `jsf-myfaces`
+  ([#7811](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7811))
+
+### 🌟 New javaagent instrumentation
+
+- R2DBC
+  ([#7977](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7977))
+- Add JFR streaming metrics gatherer
+  ([#7886](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7886))
+- ZIO 2.0 instrumentation
+  ([#7980](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7980))
+
+### 🌟 New library instrumentation
+
+- R2DBC
+  ([#7977](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7977))
+- Add JFR streaming metrics gatherer
+  ([#7886](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7886))
+- Add library instrumentation for java http client
+  ([#8138](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8138))
```

Comment:
Dupe section.

Reply:
one is for javaagent instrumentation and one is for library instrumentation (maybe we don't need to separate them anymore?)

Reply:
Oh, derp I didn't even notice that one word difference. :)